CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Added
+- Buttons StreamBlock for WhatsApp messages
+
+### Changed
+- Migrated next_prompt on WhatsApp messages to buttons
+- Removed next_prompt on WhatsApp messages
```

Comment:
I've just checked. The exporter handles `next_prompt`, but the (current) importer apparently ignores it. Merging this branch into the import/export refactor PR gives me test failures for:
* `next_prompt` not being preserved across import->export with new importer (kind of expected because it's been removed from the model, but I'd expect the export to fail because of the missing field -- apparently we explicitly ignore those errors, which we probably shouldn't)
* (empty) `buttons` field not being present in export->import with current importer (expected because the current importer builds the content blocks generically instead of using the appropriate block models)

Reply:
Yeah so import/export will need to be changed, but that's a bit difficult while it's still being worked on. I don't really want to make the changes to the old import/export, if it's going to be replaced anyway. Maybe we could hold off on this PR until that work is complete, then rebase/merge, and add to the new importer?